core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -988,7 +988,7 @@ class HttpGoogleServicesDAO(
         } else {
```

Comment:
the problem was [2 lines up from here](https://github.com/broadinstitute/rawls/pull/1131/files#diff-5ecf29f76a772ac794b30c2ba0a65dd7R986) on `newPolicies.equals(existingPolicies)`. When a `List` as in use and a duplicate entry was made to a policy the equality check would return false but then the distinct call on 991 would remove it anyway and we would call google for a noop and concurrent calls would crash all over each other. Better to just use `Set` all the way through.